name_resolution/name_resolver.rb

```diff
@@ -0,0 +1,21 @@
+require 'resolv'
+
+class NameResolver < Scout::Plugin
+  OPTIONS=<<-EOS
+    nameserver:
+      default: 8.8.8.8
```

Comment:
This isn't the nameserver we want to test against -- localhost (127.0.0.1) would be better.